features/logging/forward_logs_to_external_elasticsearch.feature

```diff
@@ -0,0 +1,149 @@
+      | file | container_json_log_template.json |
+    Then the step should succeed
+
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+    Given admin ensures "instance" cluster_log_forwarder is deleted from the "openshift-logging" project after scenario
+    And I obtain test data file "logging/clusterlogforwarder/elasticsearch/<file>"
+    When I process and create:
+      | f | <file> |
+      | p | URL=<scheme>://elasticsearch-server.<%= cb.es_proj.name %>.svc:9200 |
+    Then the step should succeed
+    And I wait for the "instance" cluster_log_forwarder to appear
+
+    Given I obtain test data file "logging/clusterlogging/fluentd_only.yaml"
+    When I create clusterlogging instance with:
+      | remove_logging_pods | true              |
+      | crd_yaml            | fluentd_only.yaml |
+    Then the step should succeed
+
+    Given I use the "<%= cb.es_proj.name %>" project
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    # check app logs
+    When I execute on the "<%= cb.log_receiver.name %>" pod:
+      | curl | -sk | -XGET | <scheme>://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"match": {"kubernetes.namespace_name": "<%= cb.proj.name %>"}}} |
+    Then the step should succeed
+    And the expression should be true> JSON.parse(@result[:stdout])['count'] > 0
+
+    # check journal logs
+    When I execute on the "<%= cb.log_receiver.name %>" pod:
+      | curl | -sk | -XGET | <scheme>://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"exists": {"field": "systemd"}}} |
+    Then the step should succeed
+    And the expression should be true> JSON.parse(@result[:stdout])['count'] > 0
+
+    # check logs in openshift* namespace
+    When I execute on the "<%= cb.log_receiver.name %>" pod:
+      | curl | -sk | -XGET | <scheme>://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"regexp": {"kubernetes.namespace_name": "openshift@"}}} |
+    Then the step should succeed
+    And the expression should be true> JSON.parse(@result[:stdout])['count'] > 0
+
+    # check audit logs
+    When I execute on the "<%= cb.log_receiver.name %>" pod:
+      | curl | -sk | -XGET | <scheme>://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"exists": {"field": "auditID"}}} |
+    Then the step should succeed
+    And the expression should be true> JSON.parse(@result[:stdout])['count'] > 0
+    """
+
+    Examples:
+      | scheme | transport_ssl_enabled | file                 |
+      | https  | true                  | clf-with-secret.yaml | # @case_id OCP-29845
+      | http   | false                 | clf-insecure.yaml    | # @case_id OCP-29846
+
+  # @author qitang@redhat.com
+  @admin
+  @destructive
+  Scenario Outline: Forward to external ES with username/password
+    Given I switch to the first user
```

Comment:
@QiaolingTang I don't think you need this statement.

Reply:
@pruan-rht I need this because I have a step to deploy logging operators before every logging scenario, see https://github.com/openshift/verification-tests/blob/master/features/support/clusterlogging_env.rb#L1, and in the [step](https://github.com/openshift/verification-tests/blob/master/features/support/clusterlogging_env.rb#L3) , I use cluster admin pseudo user, see https://github.com/openshift/verification-tests/blob/master/features/step_definitions/logging.rb#L33.